tests/std/tests/GH_000311_enforce_core_headers/test.compile.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CORE_HEADERS_ONLY

```

Comment:
Why are there two defines?

Reply:
This is still a test that passes :)

I will see how they work with failing tests in LLVM.

Reply:
https://github.com/microsoft/STL/blob/bd7adb4a932725f60ba096580c415616486ab64c/stl/inc/__msvc_all_public_headers.hpp#L76-L80